### PR TITLE
Rap3.0 & Diirt

### PR DESCRIPTION
**Pull request by jbobnar**
I updated all the RAP stuff to RAP3.0 and replaced pvmanager references with diirt (in rap features). 

Please, do not get scared of the number of files. The majority of changes in the common plugins are the result of the fact that eclipse guys diverged the RAP API from the RCP API. This broke the "single-sourcing design" for a few classes: ColorConstants, SchemeBorder and ButtonBorder. There are no functional changes to the common code.

The RAP plugins have been updated to work with RAP 3.0 and the rap.rtplot has been synchronised with the swt.rtplot.

### CODE REVIEW

File: applications/opibuilder/opibuilder-plugins/org.csstudio.opibuilder.rcp/src/org/csstudio/opibuilder/scriptUtil/ScriptUtilSSHelperImpl.java [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2016 ITER Organization.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ ******************************************************************************/
```
Comment (berryma4) [inline applications/opibuilder/opibuilder-plugins/org.csstudio.opibuilder.rcp/src/org/csstudio/opibuilder/scriptUtil/ScriptUtilSSHelperImpl.java]: ?????

File: applications/opibuilder/opibuilder-plugins/org.csstudio.opibuilder.rcp/src/org/csstudio/opibuilder/util/SingleSourceHelperImpl.java [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2016 ITER Organization.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ ******************************************************************************/
```
Comment (berryma4) [inline applications/opibuilder/opibuilder-plugins/org.csstudio.opibuilder.rcp/src/org/csstudio/opibuilder/util/SingleSourceHelperImpl.java]: ?????

**Comment by jbobnar** [comment]
Removed the ITER copyright headers which shouldn't be there. I mixed this up when merging. Thank you for noticing.

**Comment by berryma4** [comment]
You need a feature for swt.xygraph.rap

**Comment by berryma4** [comment]
Actually, bigger issue. swt.xygraph rap or rcp doesn't have a feature.
So we need two, or add it to utilities[rcp|rap], because rtplot is there.

**Pull request merged**